apis/swagger.yml

```diff
@@ -370,6 +370,29 @@ paths:
               $ref: "#/definitions/SearchResultItem"
         500:
           $ref: "#/responses/500ErrorResponse"
+      parameters:
+        - name: "term"
+          in: "query"
+          description: "Term to search"
+          type: "string"
+          required: true
+        - name: "registry"
+          in: "query"
+          description: "Search images from specified registry"
+          type: "string"
+
+        - name: "limit"
```

Comment:
the limit and filter args seems not support in this pr? could we leave a TODO or add these two args in next pr.

Reply:
right